README.md

```diff
@@ -48,6 +48,8 @@ document (consecutive objects): <wire version : int> <metadata> <page>...
 metadata (dict): {
   "Pages": int,
   "FileName": str,
+  "XFA": dict,
+  "EmbeddedFiles": [<embedfile>...],
```

Comment:
could you put `(or number of embedded files if --embedded-files not specified)`? (and likewise for `XFA`).

I didn't know about XFA before your PR, so it might be nice to have something describing what it is somewhere in the readme (particularly please use the word "form").

Reply:
Done in a separate section in README (and commit).